src/assets/i18n/en.json

```diff
@@ -256,7 +256,8 @@
             "UNKNOWN": "Unknown Error",
             "UNKNOWN_FILE_TYPE": "Unknown file type",
             "VIDEO_NOT_FOUND": "Video was not found",
-            "DUPLICATE_CHANNEL": "A channel with this name already exist"
+            "DUPLICATE_CHANNEL": "A channel with this name already exist",
+            "FILE_SIZE_TOO_BIG": "גודל הקובץ חורג מהגודל המותר"
```

Comment:
Should be ENGLISH!!!!!!!!!!!!!!!!!!!!! NOT hebrew

Reply:
Fixed